x-pack/winlogbeat/module/security/test/testdata/security-windows2012_4673.evtx.golden.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "@timestamp": "2020-04-06T06:39:04.5491199Z",
+    "event": {
+      "action": "privileged-service-called",
+      "category": "iam",
+      "code": 4673,
+      "kind": "event",
+      "module": "security",
+      "outcome": "success",
+      "provider": "Microsoft-Windows-Security-Auditing",
+      "type": "admin"
+    },
+    "host": {
+      "name": "DC_TEST2k12.TEST.SAAS"
+    },
+    "log": {
+      "level": "information"
+    },
+    "process": {
+      "executable": "C:\\Windows\\System32\\lsass.exe",
```

Comment:
We can populate `process.command_line` here too.

Reply:
Hi @andrewkroh , I did all modifications about lower camel case, spaces in assignments and between function args. Now testing the changes in my environment and then I'll commit the changes soon.
I have a doubt regarding to populate the process.command_line in event 4673.
I'm not able to see that information in the event, as for example in event 4688. Am I missing something?

![image](https://user-images.githubusercontent.com/33020901/79985470-f18bd000-84aa-11ea-968b-30ff4df7399f.png)